src/taco-dependency-installer/elevatedInstaller.ts

```diff
@@ -103,6 +104,12 @@ class ElevatedInstaller {
         this.configFile = process.argv[2];
         this.parentSessionId = process.argv[3];
         this.socketPath = process.argv[4];
+
+        // Deal with auto-accept prompts; we need to reset this value, because on Windows we launch the elevated installer through PowerShell, and doing it that way does not carry over the env of the
+        // parent node.js process
+        if (process.argv[5] === "true") {
```

Comment:
What Jimmy said; On Mac we can use "sudo -E" to keep the environment through elevation, but it doesn't look like there is something similar for Windows. So, we pass the noPrompt flag as a command line parameter, and then we set tacoGlobalConfig.noPrompt again in the elevated installer based on that